code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -137,6 +157,12 @@
 		if(M.client && ishuman(M))
 			rabid = 1
 
+		if(transformeffects & SLIME_EFFECT_GREEN)
+			visible_message("<span class='warning'>[src] slurps up [M]!</span>")
+			adjust_nutrition(10)
+			layer = initial(layer)
+			M.forceMove(src)
```

Comment:
If the slime in question doesn't barf its contents on its death (I haven't tested it but slimes probably won't; mobs having other mobs as contents generally requires special handling; see morphs and slaughter demons), then this is a bit of a very weird situation. You also can call `dust`, `gib`, or `qdel` if you want to cause permadeath.